ci/gpu/build.sh

```diff
@@ -78,6 +78,10 @@ GTEST_OUTPUT="xml:${WORKSPACE}/test-results/" make -j${PARALLEL_LEVEL} test
 logger "pip install cupy"
 pip install cupy-cuda92
 
+# Temporarily install feather for testing
+logger "conda install feather-format"
+conda install -c conda-forge -y feather-format
```

Comment:
I added conditional handling in case this isn't installed in CI, but I imagine we'd want this anyway until it's baked into the container.

Reply:
I believe you should also add this to the `cudf_dev` environment?

Reply:
This is only needed for Pandas 0.23.4 and older. If a user gets 0.24 or newer it uses pyarrow which is already a dependency. Given the exception that's thrown is pretty clear that `feather-format` needs to be installed what do you think?

Reply:
Assuming that all our docker containers are Pandas 0.24, lets leave it as is. Sounds good.